index.html

```diff
@@ -42,6 +42,56 @@ <h1 style="color: white; font: 500 30px/36.6px Google Sans"> Your Google Ads jou
+                <div class="impressions-background"></div>
+                <img src="https://fonts.gstatic.com/s/i/productlogos/google_my_business/v6/192px.svg" class="logo">
+                <span class="business-name" style="color: #000000;">MeriBindiya</span>
+                <div class="impressions-text">
+                    <span style="font-size: 0.8em; color: #5f6368;">In last </span>
+                    <span style="font-size: 1.6em; font-weight: bolder; color: #d93025;">30 days </span>
+                    <span>your ads received </span>
+                    <span style="font-size: 1.6em; font-weight: bolder; color: #d93025;">7000 users </span>
+                    <span style="font-size: 0.8em; color: #5f6368;">on </span>
+                    <span>Google Search, Maps and YouTube</span>
+                </div>
+                <div class="google-icons" style="background-image: url(image/googleg_24dp.svg); left: 70px;"></div>
+                <div class="google-icons" style="background-image: url(image/maps_24dp.svg); left: 200px;"></div>
+                <div class="google-icons" style="background-image: url(image/youtube_24dp.svg); left: 340px;"></div>
+            </div>
+        </div>
+
+        <div class="slide">
+            <div class="impressions" style="background-color: #2a2242; position: relative; ">
+                <img src="https://fonts.gstatic.com/s/i/productlogos/google_my_business/v6/192px.svg" class="logo">
+                <span class="business-name">MeriBindiya</span>
+                <h1 id="alpvisits" data-target="3000">0</h1>
+                <p>
+                    users visited your business profile
+                </p>
+                <div class="click-icon"></div>
+            </div>
+            <canvas id=confetti style="position: fixed; top: 0; left: 0;"></canvas>
+            <script src="js/alpvisits.js"></script>
+          <!--  <script src="js/confetti.js"></script> -->
+        </div>
+
+        <div class="slide">
+            <div class="impressions" style="background-color: #244d52; position: relative; ">
+                <img src="https://fonts.gstatic.com/s/i/productlogos/google_my_business/v6/192px.svg" class="logo">
+                <span class="business-name">MeriBindiya</span>
+                <div class="icons group1"><img src="image/call.png" style="max-height: 100%;"></div>
+                <div class="icons group2"><img src="image/gmb.png" style="max-height: 100%;"></div>
+                <div class="icons group3"><img src="image/list.png" style="max-height: 100%;"></div>
+                <div class="content count call-count">200</div>
+                <div class="content text call-text">calls</div>
+                <div class="content count visit-count">150</div>
+                <div class="content text visit-text">store visits</div>
+                <div class="content count lead-count">100</div>
+                <div class="content text lead-text">submitted leads</div>
```

Comment:
What are submitted leads in AdsGo context?

Reply:
form fills